src/main/resources/META-INF/services/com.fasterxml.jackson.databind.Module

```diff
@@ -38,3 +38,28 @@ org.killbill.billing.util.tag.boilerplate.Module
 org.killbill.billing.overdue.api.boilerplate.Module
 org.killbill.billing.catalog.api.boilerplate.Module
 
+#  Generated by https://github.com/killbill/api-pojos. Do not edit!
```

Comment:
Only 1: The copyright header generated twice, but I can't reproduce this :( . I think the best is just to remove the header/copyright manually. WDYT @pierre @sbrossie 

This is generated `com.fasterxml.jackson.databind.Module` file on my local (without edit anything manually):

```
#  Generated by https://github.com/killbill/api-pojos. Do not edit!
#  
#  Copyright 2022-2022 The Billing Project, LLC
#  
#  The Billing Project licenses this file to you under the Apache License,
#  version 2.0 (the "License"); you may not use this file except in compliance
#  with the License.  You may obtain a copy of the License at:
#  
#     http://www.apache.org/licenses/LICENSE-2.0
#  
#  Unless required by applicable law or agreed to in writing, software
#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
#  License for the specific language governing permissions and limitations
#  under the License.

org.killbill.billing.util.customfield.boilerplate.Module
org.killbill.billing.security.api.boilerplate.Module
org.killbill.billing.osgi.api.config.boilerplate.Module
org.killbill.billing.invoice.api.boilerplate.Module
org.killbill.billing.currency.api.boilerplate.Module
org.killbill.billing.util.api.boilerplate.Module
org.killbill.billing.invoice.api.formatters.boilerplate.Module
org.killbill.billing.tenant.api.boilerplate.Module
org.killbill.billing.boilerplate.Module
org.killbill.billing.usage.api.boilerplate.Module
org.killbill.billing.catalog.api.rules.boilerplate.Module
org.killbill.billing.util.audit.boilerplate.Module
org.killbill.billing.util.nodes.boilerplate.Module
org.killbill.billing.payment.api.boilerplate.Module
org.killbill.billing.account.api.boilerplate.Module
org.killbill.billing.util.entity.boilerplate.Module
org.killbill.billing.payment.plugin.api.boilerplate.Module
org.killbill.billing.entitlement.api.boilerplate.Module
org.killbill.billing.osgi.api.boilerplate.Module
org.killbill.billing.util.callcontext.boilerplate.Module
org.killbill.billing.util.tag.boilerplate.Module
org.killbill.billing.overdue.api.boilerplate.Module
org.killbill.billing.catalog.api.boilerplate.Module

org.killbill.billing.payment.plugin.api.boilerplate.plugin.Module
org.killbill.billing.catalog.plugin.api.boilerplate.plugin.Module
org.killbill.billing.entitlement.plugin.api.boilerplate.plugin.Module
org.killbill.billing.notification.plugin.api.boilerplate.plugin.Module
org.killbill.billing.control.plugin.api.boilerplate.plugin.Module
org.killbill.billing.invoice.plugin.api.boilerplate.plugin.Module
org.killbill.billing.currency.plugin.api.boilerplate.plugin.Module
```

Reply:
> I think the best is just to remove the header/copyright manually
👍 Maybe we can come up with a simple one-liner script (using maybe `awk` or `sed`)?

Reply:
> Maybe we can come up with a simple one-liner script 

But this is on @reshmabidikar side, and I'm not sure what is `awk`/`sed` command in windows. So, personally I suggest @reshmabidikar edit the file manually. I submit [new issue here](https://github.com/killbill/api-pojos/issues/16) to track this problem. I just leave it as is and not started work on that, unless someone ask me to fix it.

Reply:
Yup, I'll fix this 👍